builders/pallets-precompiles/precompiles/author-mapping.md

```diff
@@ -0,0 +1,148 @@
+---
+title:  Author Mapping Precompile
+description: A guide for collators to learn how to use the author mapping solidity interface to map session keys to a Moonbeam address where block rewards are paid out. 
+keywords: solidity, ethereum, author mapping, collator, moonbeam, precompiled, contracts, block producer
+---
+
+# Interacting with the Author Mapping Precompile
+
+![Precomiled Contracts Banner](/images/builders/pallets-precompiles/precompiles/author-mapping/author-mapping-banner.png)
+
+## Introduction {: #introduction } 
+
+The author mapping precompiled contract on Moonbeam allows collator candidates to map session keys to a Moonbeam address, where block rewards are paid out, through a familiar and easy-to-use Solidity interface.
```

Comment:
Should we say that this allows them to run a collator using Ledger?

Reply:
It's in the following paragraphs - should I move it up to this first sentence/paragraph?

Reply:
Sure. I feel we should show the main value of this precompile a bit earlier